include/slam_toolbox/visualization_utils.hpp

```diff
@@ -54,16 +55,25 @@ inline visualization_msgs::msg::Marker toMarker(
 
   return marker;
 }
+template<class NodeT>
+inline visualization_msgs::msg::Marker toMarker(
+  const std::string & frame,
+  const std::string & ns,
+  const double & scale,
+  NodeT && node)
+{
+  return toMarker(frame, ns, scale, node->get_clock());
```

Comment:
Why not just pass the NodeT? Seems silly to duplicate the function just to get the clock to pass to another

Reply:
It was needed to call from LoopClosureAssistant which does not have a NodePtr (at this time).
Also, I wanted to keep the API with NodePtr just in case. 

I will about this again after <https://github.com/SteveMacenski/slam_toolbox/pull/561#discussion_r1049514461>

<https://github.com/SteveMacenski/slam_toolbox/blob/78af8485ebfd7d2b95e50188a341b86b4069e644/src/loop_closure_assistant.cpp#L216>

<https://github.com/SteveMacenski/slam_toolbox/blob/78af8485ebfd7d2b95e50188a341b86b4069e644/src/merge_maps_kinematic.cpp#L143-L144>

Reply:
In order to handle `rclcpp::Node::SharedPtr` and `rclcpp::Clock::SharedPtr` in common, I have updated the template name to `ClockHandler` and remove duplicate function.

<https://github.com/SteveMacenski/slam_toolbox/blob/0b8ad824d5e544425bc08bfc629a213b032ca9d0/include/slam_toolbox/visualization_utils.hpp#L62-L72>